kod/include/blakston.khd

```diff
@@ -2154,7 +2154,7 @@
    PFLAG2_DETECT_GOOD      = 0x000004
    PFLAG2_DETECT_EVIL      = 0x000008
    PFLAG2_DETECT_INVIS     = 0x000010
-   PFLAG2_PARALYZED        = 0x000020
+   PFLAG2_PARALYZED        = 0x000020 % Not currently used by anything
```

Comment:
Delete this flag, which will prove that it really isn't used anywhere else.

It would be a good idea to leave a comment here identifying that the value 0x00020 is currently unused.